server/src/main/java/org/elasticsearch/index/query/FieldMaskingSpanQueryBuilder.java

```diff
@@ -152,7 +152,7 @@ public static FieldMaskingSpanQueryBuilder fromXContent(XContentParser parser) t
     @Override
     protected SpanQuery doToQuery(QueryShardContext context) throws IOException {
         String fieldInQuery = fieldName;
-        MappedFieldType fieldType = context.fieldMapper(fieldName);
+        MappedFieldType fieldType = context.getFieldType(fieldName);
         if (fieldType != null) {
             fieldInQuery = fieldType.name();
         }
```

Comment:
Basically everywhere that does this null check should be calling `isMapped` instead.